buttonHandler.js

```diff
@@ -49,19 +66,23 @@ ButtonHandler.prototype.isTrue = function(target, eventCount, mainWindow) {
 	this.statsIsTrueInvokedCount++;
 	var result;
 	if (this.optionalIsTrueFunction == null) {
+		if (this.debugLogElementTreeOnIsTrue) {
+		 UIATarget.localTarget().logElementTree();
+	    }
 		var aButton = this.findButton(target);
-        result = aButton.isNotNil() && aButton.validAndVisible();
-    } else {
-	    result = this.optionalIsTrueFunction(target, eventCount, mainWindow);
-    }
-    if (result) {
-	  this.statsIsTrueReturnedTrue++;
-    } else {
-	  this.statsIsTrueReturnedFalse++;
-    };
-    return result;
+		result = this.isValidAndVisible(aButton);
```

Comment:
Sigh, the earlier version had some cruft in it-some spaces instead of tabs on line 71, and a leading space instead of a tab on line 70. The newer version does show more lines changed, but I believe this is a good thing as the formatting is now improved.